QuotesRotator/css/main.css

```diff
@@ -1,134 +1,124 @@
-/*! HTML5 Boilerplate v5.1.0 | MIT License | https://html5boilerplate.com/ */
-
-/*
+  /*! HTML5 Boilerplate v5.1.0 | MIT License | https://html5boilerplate.com/ */
+  /*
  * What follows is the result of much research on cross-browser styling.
  * Credit left inline and big thanks to Nicolas Gallagher, Jonathan Neal,
  * Kroc Camen, and the H5BP dev community and team.
  */
-
-/* ==========================================================================
+  /* ==========================================================================
    Base styles: opinionated defaults
    ========================================================================== */
-
-html {
+  
+  html {
     color: #222;
     font-size: 1em;
     line-height: 1.4;
-}
-
-/*
+  }
+  /*
  * Remove text-shadow in selection highlight:
  * https://twitter.com/miketaylr/status/12228805301
  *
  * These selection rule sets have to be separate.
  * Customize the background color to match your design.
  */
-
-::-moz-selection {
+  
+  ::-moz-selection {
     background: #b3d4fc;
     text-shadow: none;
-}
-
-::selection {
+  }
+  ::selection {
     background: #b3d4fc;
     text-shadow: none;
-}
-
-/*
+  }
+  /*
  * A better looking default horizontal rule
  */
-
-hr {
+  
+  hr {
     display: block;
     height: 1px;
     border: 0;
     border-top: 1px solid #ccc;
     margin: 1em 0;
     padding: 0;
-}
-
-/*
+  }
+  /*
  * Remove the gap between audio, canvas, iframes,
  * images, videos and the bottom of their containers:
  * https://github.com/h5bp/html5-boilerplate/issues/440
  */
-
-audio,
-canvas,
-iframe,
-img,
-svg,
-video {
+  
+  audio, canvas, iframe, img, svg, video {
     vertical-align: middle;
-}
-
-/*
+  }
+  /*
  * Remove default fieldset styles.
  */
-
-fieldset {
+  
+  fieldset {
     border: 0;
     margin: 0;
     padding: 0;
-}
-
-/*
+  }
+  /*
  * Allow only vertical resizing of textareas.
  */
-
-textarea {
+  
+  textarea {
     resize: vertical;
-}
-
-/* ==========================================================================
+  }
+  /* ==========================================================================
    Browser Upgrade Prompt
    ========================================================================== */
-
-.browserupgrade {
+  
+  .browserupgrade {
     margin: 0.2em 0;
     background: #ccc;
     color: #000;
     padding: 0.2em 0;
-}
-
-/* ==========================================================================
+  }
+  /* ==========================================================================
    Author's custom styles
    ========================================================================== */
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-/* ==========================================================================
+  
+  body {
+    width: 100%;
+  }
+  .container {
+    width: 30%;
+    height: 30%;
```

Comment:
Rather than an explicit `height` property, what about `min-height`? Also, what is `height` 30% of...?